README.md

```diff
@@ -252,7 +252,7 @@ ActiveAdmin.register Page do
     actions
   end
 
-  form do |f|
+  form html: { class: "edit_atomic_content" } do |f|
```

Comment:
FWIW, I think the ids are being established by the form helper and that is why they have underscores.